.circleci/config.yml

```diff
@@ -132,46 +200,87 @@ jobs:
       QEMU_ARCH: aarch64
       TAG: arm64v8
       TARGET: arm64v8
+  build-arm32v7:
+    <<: *shared
+    environment:
+      QEMU_ARCH: arm
+      TAG: arm32v7
+      TARGET: arm32v7
 
 workflows:
   version: 2
   build_and_test:
     jobs:
-      - test-server
-      - test-front
+      - test-server:
+          filters:
+            tags:
+              only: /^v\d+.\d+.\d+$/
+            branches:
+              only: master
+      - test-front:
+          filters:
+            tags:
+              only: /^v\d+.\d+.\d+$/
+            branches:
+              only: master
       - build-front:
           requires:
             - test-server
             - test-front
+          filters:
+            tags:
+              only: /^v\d+.\d+.\d+$/
+            branches:
+              only: master
       - build-armhf:
           requires:
             - build-front
           filters:
             tags:
-              only: /v*/
+              only: /^v\d+.\d+.\d+$/
             branches:
               only: master
```

Comment:
For all the build jobs, anchors and aliases might help to avoid repeating the same content n times ? Something like : 
```yaml
build: &build
   requires:
     - build-front
   filters:
   tags:
     only: /v*/
     only: /^v\d+.\d+.\d+$/
   branches:
     only: master

- build-armhf:
  <<: *biuld
```

Reply:
Definitely! That's what we are doing actually no @VonOx ?

Reply:
Yes we use anchors but not for filtering, I'm open on this because filters not arch specific